react/util/ui-utils.js

```diff
@@ -2,7 +2,7 @@ import { Alert, Linking, Platform, ToastAndroid } from "react-native";
 import DocumentPicker from "react-native-document-picker";
 import ImagePicker from "react-native-image-picker";
 
-const normalizeAttachment = function (attachment) {

```

Comment:
@Benedito0 when this kind of things happen, the way to go is opening a separate PR to address it (see https://github.com/ripe-tech/ripe-components-react-native/pull/135). Otherwise you have a PR that solves more than one thing, which highly difficulties the task of managing dozens of PRs per day.

Reply:
In this case, you're using outdated dependencies. `npm i` + `npm run pretty` and push, to get rid of this diff.